server/src/organizations/entities/organization.entity.ts

```diff
@@ -41,9 +41,9 @@ export class Organization {
   @OneToMany(() => UserOrganization, (user_org) => user_org.organization)
   users: UserOrganization[];
 
-  @OneToMany(
-    () => Transaction,
-    (transaction) => transaction.donater_organization || transaction.recipient,
-  )
-  transactions: Transaction[];
+  @OneToMany(() => Transaction, (transaction) => transaction.donater_organization)
+  donated_transactions: Transaction[];
+
+  @OneToMany(() => Transaction, (transaction) => transaction.claimer)
+  claimed_transactions: Transaction[];
 }
```

Comment:
Split the two to make them more explicit. One tracks donated stuff, the other claimed stuff